learn/tanssi/technical.md

```diff
@@ -3,51 +3,83 @@ title: Technical Features
+![Collators Assignment Algorithm](/images/learn/tanssi/technical/light-technical-1.png#only-light)
+![Collators Assignment Algorithm](/images/learn/tanssi/technical/dark-technical-1.png#only-dark)
 
+## Block Production {: #block-production } 
 
-## Main features
+Polkadot is a heterogeneous multi-chain ecosystem, where multiple parallel blockchains connect to a central blockchain called the relay chain. To provide block production as a service, Tanssi Protocol masterfully orchestrates and designs the interaction between Tanssi Network, the ContainerChains, and Polkadot's relay chain.
 
-Polkadot is layer 0 blockchain that offers essential services to other chains within the ecosystem, including security, consensus and interoperability/communication, leaving the implementation of specific use cases to the parachains while tackling these common problems:
+In the Polkadot ecosystem, the Appchains connected to the relay chain are fully sovereign blockchains, having their own rules, consensus mechanisms, and so forth, and this is the case for Tanssi Network and the Appchains deployed through Tanssi as well. 
 
-1. Scalability: Polkadot's sharded model allows for the parallel processing of transactions across multiple parachains, significantly improving scalability and overall network capacity while lowering processing times and fees. 
-2. Interoperability: While most blockchain networks operate in isolation, making it difficult for them to communicate and share assets, Polkadot provide native interoperability through XCM (Cross Consensus Message Format), connecting multiple chains facilitating asset transfer capabilities and information exchange.
-3. Governance and Upgrades: Polkadot introduces an on-chain governance model, empowering token holders to participate in decision-making and allowing for efficient upgrades and protocol improvements.
-4. Security: In smaller blockchain networks, security can be a concern due to lower levels of mining or staking participation. Polkadot’s model of shared security guarantees the same high security level even for the smaller parachain within the ecosystem.
-5. Customizability/flexibility: Polkadot's architecture encourages the creation of sovereign blockchains with high specialization, enabling developers to design chains with specific features and functionalities optimized for the unique requeriments of the use case.
+We can consider Tanssi Network and the ContainerChains as sibling chains, meaning that there is no hierarchical dependency whatsoever, and we will discuss their responsibility and how they interact with each other through the relay chain in the following sections.
```

Comment:
```suggestion
The Tanssi network and the ContainerChains can be considered as sibling chains, meaning that there is no hierarchical dependency whatsoever. Their responsibility and how they interact with each other through the relay chain will be covered in the following sections.
```

Try to avoid using "we" or "let's" unless it's an informal writing piece like a tutorial